src/index.css

```diff
@@ -109,4 +109,12 @@ select {
   padding-top: 20px;
 }
 
+.build-icon{
+  color: rgb(218, 39, 39);
+}
```

Comment:
This would probably be better named something like: "active-build-icon"

Reply:
I updated how the icons are rendered and am using Mui's color prop instead. This has been removed